commissioning.tex

```diff
@@ -1,27 +1,50 @@
 \section{Rubin Observatory Commissioning}
 \label{sec:commissioning}
 
+\subsection{Commissioning Milestones}
+\label{ssec:commissioning-milestones}
+
+Commissioning work is being planned around two major milestones, ''First Photon`` and ''System First Light``. 
+
+\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the LSSTCam focal plane to produce an image of the night sky.
+
+\textbf {System First Light}: Defined as the first science-grade images of the night sky obtained with LSSTCam. 
```

Comment:
We have been working on a more detailed definition of System First Light https://sitcomtn-061.lsst.io/v/u-kbechtol-init/index.html

Our working definition has been that System First Light is achieved when we can "routinely acquire science-grade imaging across the full focal plane, and have a well understood technical path towards meeting the Construction Completeness criteria"

Reply:
Integrated into text